css/index.css

```diff
@@ -131,16 +326,122 @@ footer {
     background: black;
 }
 
-footer nav {
-    width: 60%;
+.ft {
     display: flex;
-    justify-content: space-between;
-    align-items: center;
-    padding: 20px 2%;
+    width: 100%;
+    height: 100%;
+    padding: 20px 20%;
     font-size: 14px;
 }
 
-footer nav a {
+.ft a {
```

Comment:
like we discussed the other day, you may want to be more explicit with your class names, because it becomes difficult to follow, especially if you come back to your code at a later time.